src/plotman/archive.py

```diff
@@ -117,26 +116,25 @@ def archive(dir_cfg, all_jobs):
     #
     # Pick first archive dir with sufficient space
     #
-    archdir_freebytes = get_archdir_freebytes(arch_cfg)
+    archdir_freebytes = get_archdir_freebytes(dir_cfg.archive)
     if not archdir_freebytes:
         return(False, 'No free archive dirs found.')
     
     archdir = ''
     available = [(d, space) for (d, space) in archdir_freebytes.items() if 
                  space > 1.2 * plot_util.get_k32_plotsize()]
-    if len(available):
-        index = arch_cfg['index'] if 'index' in arch_cfg else 0
-        index = min(index, len(available) - 1)
+    if available:
```

Comment:
I'm not requesting a change here, but I would have gone the other way if anything.  `if len(available) > 0:`  In this case it doesn't really matter since it is necessarily a list and that is forced on the line above.  But, I prefer explicitness in tests.  There was a case recently in plotman where a 0 was misprocessed the same as a `None`, IIRC.  Anyways, opinions etc.  Just sharing mine so we can all learn each other's tendencies.  `:]`

Reply:
Given that `available` is produced via a list comprehension, it's safe from `None` issues, but I'm totally happy to change to `if len(available) > 0`. My preference is either my original code or what you suggested instead of `if len(available)` :)